R/epi_df.R

```diff
@@ -111,6 +111,51 @@ NULL
 #' @return An `epi_df` object.
 #'
 #' @export
+#' @examples 
+#' # Convert a `tsibble` that has county code as an extra key
+#' ex1 <- tsibble::tibble(
```

Comment:
Here and below, `tsibble::tibble` -> `tibble::tibble`.  Too easy to read `tsibble::` and think this is `tsibble::tsibble`.